src/Styleguide/Pages/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -39,10 +28,10 @@ export class Commercial extends React.Component<CommercialProps> {
     return editions.map((edition, index) => {
       return (
         <React.Fragment key={edition.__id}>
-          <PricingInfoContainer pb={2}>
```

Comment:
We specifically kept this redirection to keep the code here more reasonable. My hope was that the browser would also render the component names in the html which make it much easier to debug. But that is not the case - so I guess does not really matter much.

Reply:
If you want helpful names in the react-devtools then you’ll need to create a new component (rather than just referring to another one such as `Box`) and give it a `displayName` (something which the styled-components babel plugin does for us).